js1/dom-innerhtml/cvlekce/novinky/exercise.md

```diff
@@ -11,7 +11,7 @@ Vytvořte si repozitář ze šablony [cviceni-novinky](https://github.com/Czechi
 1. Do jiné proměnné vyberte element s třídou `news` a nastavte mu barvu pozadí na bílou a maximální šířku na `60rem`.
 1. Vyberte element `h1` a nastavte mu (v JavaScriptu) třídu na `news__title`. Nadpis by měl změnit styl. Nastavte také obsah nadpisu na text `Aktuální novinky`.
 1. Pomocí atributu `id` vyberte element obsahující první zprávu. Přidejte do jeho atributu `class` třídu `post--main`. První zpráva by tak měla mírně změnit svůj styl.
-1. Vyberte obrázek v poslední zprávě (tu vyberte podle `id`) a změnte jeho atribut `src` na obrázek `images/zprava3-novy.jpg`.
+1. Pomocí CSS selektoru vyberte obrázek ve třetí zprávě (`id` zprávy je `zprava3`) a změňte jeho atribut `src` na obrázek `img/zprava3-novy.jpg`.
```

Comment:
Proč `img`? V tom repozitáři jsou to `images`.

Reply:
Vedlejší otázka je, jestli se shodujeme na tom, že zkratkám se vyhýbáme?

Reply:
> Proč `img`? V tom repozitáři jsou to `images`.

`img` byla chyba zavlečená z nějaké staré větve. Opraveno.